src/editor-sidebar/about.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalText as Text,
+	PanelBody,
+	ExternalLink,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import ScreenHeader from './screen-header';
+
+function AboutPlugin() {
+	return (
+		<PanelBody>
+			<ScreenHeader
+				title={ __( 'About the plugin', 'create-block-theme' ) }
+			/>
+			<VStack>
+				<Text>
+					{ __(
+						'Create Block Theme is a tool to help you make Block Themes using the WordPress Editor. It does this by adding tools to the Editor to help you create and manage your theme.',
+						'create-block-theme'
+					) }
+				</Text>
+
+				<Text>
+					{ __(
+						"Themes created with Create Block Theme don't require Create Block Theme to be installed on the site where the theme is used.",
+						'create-block-theme'
+					) }
+				</Text>
+
+				<hr />
```

Comment:
It looks like we use a `CardDivider` rather than a `hr` in Gutenberg, I think we should use them here as well.

Reply:
Oh I see, looks good!